test/mock/rtps/WriterProxyData/rtps/builtin/data/WriterProxyData.hpp

```diff
@@ -52,7 +52,7 @@ class WriterProxyData : public PublicationBuiltinTopicData
         , security_attributes_(0)
         , plugin_security_attributes_(0)
 #else
-            : m_network_configuration(0)
+        , m_network_configuration(0)
```

Comment:
We can just remove these two lines, since `m_network_configuration` is now on `PublicationBuiltinTopicData`

Reply:
IDK if `loopback_transformation` within `PublicationBuiltinTopicData` is meant to replace `m_network_transformation`  or not (probably we can remove the `m_network_transformation`) . Anyways, in the mock is not needed to have it as a class member so I will remove it.